web/react-web/src/Sidebar/index.js

```diff
@@ -23,88 +21,75 @@ class Sidebar extends Component {
 
     render() {
         return (
-            <AvailablePredictionsContext.Consumer>
-                {context =>
-                    <GraphDetailContext.Consumer>
-                        {graphDetailContext => {
-                            const dates = [];
-                            const {isXAxisRelative, axesType} = graphDetailContext.options;
-                            const {activeTab} = this.state;
-
-                            for (let key in context.data) {
-                                const item = context.data[key];
-                                const date = new Date(item.label);
-                                const className = `tab${this.state.activeDateKey === key ? ' active' : ''}`;
+            <GraphDetailContext.Consumer>
+                {graphDetailContext => {
+                    const {isXAxisRelative, axesType} = graphDetailContext.options;
+                    const {activeTab} = this.state;
 
-                                dates.push(<button key={key} className={className} onClick={this.selectTab(key)}>{date.toLocaleDateString(undefined, options)}</button>);
-                            }
-
-                            return (
+                    return (
+                        <>
+                            <div className='buttons'>
+                                <button className={activeTab === 0 ? 'active' : ''} onClick={this.selectTab(0)}>About</button>
+                                <button className={activeTab === 1 ? 'active' : ''} onClick={this.selectTab(1)}>Results</button>
+                            </div>
+                            {activeTab === 0 &&
+                                //TODO: load markdown content from backend and render it, once it'll be available
+                                <>
+                                    <h1>COVID-19 predictions of Boďová and Kollár</h1>
+                                    <p>Mathematicians Katarína Boďová and Richard Kollár predicted in March and April 2020
```

Comment:
By the way, we have clarified this text recently in the Python code base. We should share the text between the projects, I think Python can provide the HTML via a REST call. Can be done in another PR.

Reply:
Either markdown or html. No problem.